pkg/sdk/tasks_impl_gen.go

```diff
@@ -65,6 +67,48 @@ func (v *tasks) Execute(ctx context.Context, request *ExecuteTaskRequest) error
 	return validateAndExec(v.client, ctx, opts)
 }
 
+// TODO(SNOW-1277135): See if depId is necessary or could be removed
```

Comment:
depId is not there anymore, but we cal live with this (correct in in some other PR please)

Reply:
Will do in the next pr.